simulator/src/simulator/simulation-node.ts

```diff
@@ -31,21 +32,35 @@ import {
   supportedNodeChildProcesses,
 } from './events'
 import { sleep } from './misc'
-import { getLatestBlockHash } from './utils/chain'
+import { getLatestBlockHash, importAccount } from './utils'
 
 export const rootCmd = 'ironfish'
 
+/**
+ * Additional configuration options for the node. These are not part of the `ConfigOptions` interface
+ */
```

Comment:
Why have a separate type for these? why not just make them optional and set sane defaults when constructing?